qutebrowser/config/configdata.yml

```diff
@@ -746,6 +746,12 @@ content.xss_auditing:
     Suspicious scripts will be blocked and reported in the inspector's
     JavaScript console.
 
+content.audible_default:
```

Comment:
@jgkamat Do you prefer `audible_default` as a name to `mute` what is what @cauimsouza used before? I'm worried about two things here:

- "audible" might not be very clear to non-native english speakers, compared to "mute" (or "muted")
- Do we really need the "_default" suffix? If I set, say, `content.muted` to `True` for a URL, I expect it to be muted (by default) - sure, I can still unmute it, but I'm not sure this needs to be reflected in the option name.

@cauimsouza what's your opinion on this?

Reply:
I agree, "content.mute" is much clearer and better comments can explain that it works for all URLs, although allowing the user to set it differently for specific URLs.